LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (C) 2015 President and Fellows of Harvard College
+Copyright (C) 2016 Python LTI Initiative
```

Comment:
That makes sense. Can we make the first line:

```
Copyright (C) 2011-2016 Top Hat Monocle Corp. and others
```

@amackera, given that wrote in the original copyright statement, would you be able to give any thoughts on the appropriate way to handle this?